### PR TITLE
Remove reference to non-existent function

### PR DESCRIPTION
**Pull request by steveno**
Only comments were modified.

### CODE REVIEW

**Comment by splinterofchaos** [comment]
LGTM :+1:

File: src/nvim/sha256.c [hunk 1]
```diff
@@ -8,8 +8,6 @@
 /// at your choice.
 ///
 /// Vim specific notes:
-/// Functions exported by this file:
-///  2. sha2_seed() generates a random header.
 /// sha256_self_test() is implicitly called once.
 
 #include <stddef.h>        // for size_t
```
Comment (Pyrohh) [inline src/nvim/sha256.c]: The `stdlib.h` and `nvim/os/time.h` includes also seem unused.

Comment (steveno) [inline src/nvim/sha256.c]: I didn't check that. I can amend the commit and resubmit.

Comment (Pyrohh) [inline src/nvim/sha256.c]: > I didn't check that

It's fine, it's just that this commit is tiny so we might as well add some minor related cleanups.

Anyways, there's no need to resubmit: just use `git push --force`.

**Comment by steveno** [comment]
I have pushed a new single commit that removes the unused comments along with the unused headers as well.

**Comment by justinmk** [comment]
LGTM. Thanks!

**Comment by Pyrohh** [comment]
Thanks both, merged.

**Pull request closed without merging**